source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/lib/index.ts

```diff
@@ -35,6 +35,18 @@ export interface EventbridgeToLambdaProps {
    * @default - Default props are used
    */
   readonly lambdaFunctionProps?: lambda.FunctionProps,
+  /**
+   * Existing instance of a custom EventBus, uses `Default` EventBus if this property is not set.
```

Comment:
Drop 'uses Default EventBus if this property is not set' as it is not true. It could create a custom event bus.

Reply:
updated